########################################
# Messages
########################################
bool moving
bool moving_forward
bool moving_backward
bool moving_imu
bool moving_scan
bool moving_forward_scan
bool moving_backward_scan
bool moving_forward_wheels
bool moving_backward_wheels
bool collision_forward
bool collision_backward
bool collision
float32 collision_avg
